submissions/EmmanuelUdoh/phpIntro/php/create.php

```diff
@@ -0,0 +1,59 @@
+<?php
+session_start();
```

Comment:
You do not have to use sessions. Fetch your brands directly here, please 🙏. You will always have to hit fetch.php here and I can't see you doing that. Instead just query the DB here. Like you do in overview.php